HW_6/HW_7.py

```diff
@@ -0,0 +1,175 @@
+
```

Comment:
After the 1st article letters_count.csv and words_count.csv are empty.

![image](https://user-images.githubusercontent.com/34323554/109439193-04115700-7a36-11eb-938d-d670c2d2d749.png)
![image](https://user-images.githubusercontent.com/34323554/109439231-2b682400-7a36-11eb-8025-c9f16105bd43.png)

Reply:
Also, data in letters_count.csv and words_count.csv are always outdated, because calculations doesn't process for last news.

The task was "CSVs should be recreated each time new record added".

For example:

![image](https://user-images.githubusercontent.com/34323554/109439369-b517f180-7a36-11eb-86e7-dde2f2a13dcc.png)

![image](https://user-images.githubusercontent.com/34323554/109439386-c2cd7700-7a36-11eb-98ef-13cc4a97daa8.png)
As can you see, the last news is not processed (we don't have "lorem" or "ipsum" words)